src/uct/ib/base/ib_md.c

```diff
@@ -583,13 +593,13 @@ static ucs_status_t uct_ib_dereg_mr(struct ibv_mr *mr)
     return UCS_OK;
 }
 
-static ucs_status_t uct_ib_memh_dereg(uct_ib_mem_t *memh)
+static ucs_status_t uct_ib_memh_dereg(uct_ib_md_t *md, uct_ib_mem_t *memh)
 {
     ucs_status_t s1, s2;
 
     s1 = s2 = UCS_OK;
     if (memh->flags & UCT_IB_MEM_FLAG_ATOMIC_MR) {
-        s2 = uct_ib_dereg_mr(memh->atomic_mr);
+        s2 = md->ops->dereg_atomic_key(md, memh);
```

Comment:
why don't we define a wrapper, like `uct_ib_reg_atomic_key`? Would be consistent with other funcs

Reply:
ok, ic